### PR TITLE
Update content

### LINKED ISSUE #7: Update Resource Page
I feel like the resources page should be redone to highlight everything we have more. The EDI and general feedback surveys are buried, there’s no reason for you to see them. This could be a design change, reordering things, or both

### PR DESCRIPTION
**Pull request by marcus-k**
Resolves #7 by highlighting the different sections and bringing focus to the links on the page.

### CODE REVIEW

**Comment by marcus-k** [comment]
Also this added a download button as suggested in #2 to the election norms page.

File: html/elecnorms.html [hunk 1]
```diff
@@ -136,6 +136,11 @@
                     <div class="stuff col-12 outline">
                         <div class="con innerStuff">
                             <embed class="pdf" type="application/pdf" src="../assets/pdfs/elecnorms.pdf"></embed>
+                            <a href="../assets/pdfs/elecnorms.pdf">
```
Comment (awsmit07) [inline html/elecnorms.html:139]: This looks good. Should we also do this for the constitution and change the pages here to close #6?

**Review by awsmit07 — CHANGES_REQUESTED** [review]

File: html/elecnorms.html (hunk 1 above)
Comment (marcus-k) [reply html/elecnorms.html:139]: This just adds the download button to the document. I can add it to the constitution page as well. #6 deals more with merging the two pages which I haven't looked into yet.

Comment (awsmit07) [reply html/elecnorms.html:139]: I think we can leave the constitution for when we do #6 then.

**Review by awsmit07 — APPROVED** [review]

File: html/elecnorms.html (hunk 1 above)
Comment (marcus-k) [reply html/elecnorms.html:139]: Sounds good, we'll do it in a separate PR then.

**Pull request merged**